lang/lua-copas/Makefile

```diff
@@ -8,18 +8,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lua-copas
-PKG_VERSION:=2.0.0
+PKG_VERSION:=2_0_2
```

Comment:
I'm still a little concerned by this version style, are you sure it plays well withopkg-compare-versoins?

Reply:
Beats me. I have no plans to backported this to 17.01 or 18.06.

Reply:
Upstream tags it like this in git. I could open an issue.